expression/expr_to_pb.go

```diff
@@ -491,5 +491,11 @@ func (pc pbConverter) convertToPBExpr(expr *ScalarFunction, tp tipb.ExprType) *t
 		}
 		children = append(children, pbArg)
 	}
+	if pc.client.IsRequestTypeSupported(kv.ReqTypeDAG, kv.ReqSubTypeSignature) {
+		code := expr.Function.PbCode()
+		if code > 0 {
+			return &tipb.Expr{Tp: tipb.ExprType_ScalarFunc, Sig: &code, Children: children, FieldType: toPBFieldType(expr.RetType)}
```

Comment:
`code` is a `int32` value, it's not suitable to escape this local variable to be allocated on heap and keep a reference of it. we can change the pb or let `Sig` point to a global variable.

Reply:
you are right. I will change this to a value in another pr